interop/src/server.rs

```diff
@@ -139,7 +139,7 @@ impl TestService for InteropTestService {
                 if req.has_response_status() {
                     let code = req.get_response_status().get_code();
                     let msg = Some(req.take_response_status().take_message());
-                    let status = RpcStatus::new(RpcStatusCode::new(code), msg);
+                    let status = RpcStatus::new(code.into(), msg);
```

Comment:
How about changing the args in new to accept `impl Into<RpcStatusCode>`?

Reply:
Done